package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "express",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "This project involves creating a RESTful server for storing, retrieving, updating, and deleting data in a database.",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "express-graphql": "^0.12.0",
```

preferimos fixar as versões de dependência para evitar possíveis erros no código com atualização de novas versões
```suggestion
    "express-graphql": "0.12.0",
```